Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -340,9 +342,8 @@ namespace ROS2
             AZ_TracePrintf("URDF Importer", "TransformComponent not found in created entity\n") return;
```

Comment:
While you are editing this file (and others) can you replace all `AZ_TracePrintf` with `AZ_Trace`?